app/assets/javascripts/discourse/app/components/discourse-topic.js

```diff
@@ -23,8 +23,6 @@ export default Component.extend(
       "topic.is_warning",
       "topic.category.read_restricted:read_restricted",
       "topic.deleted:deleted-topic",
-      "topic.categoryClass",
-      "topic.tagClasses",
```

Comment:
Instead we're using the [`add-category-tag-classes`](https://github.com/discourse/discourse/blob/42f6c9b6b9db81e2a6a9ca45e76f067980525f67/app/assets/javascripts/discourse/app/components/add-category-tag-classes.js) component.